src/main/java/org/opentripplanner/graph_builder/module/osm/OSMDatabase.java

```diff
@@ -50,7 +50,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class OSMDatabase {
+public class OSMDatabase implements org.opentripplanner.openstreetmap.spi.OSMDatabase {
```

Comment:
What does SPI stand for here?

Reply:
[Service provider interface](https://en.wikipedia.org/wiki/Service_provider_interface)